<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="com.testcase.LoginDemoApp" tests="4" time="0.001" ignored="0" failures="0" timestamp="11 Feb 2021 23:24:45 GMT" skipped="3" hostname="Dell" errors="1">
  <testcase name="Demo" time="0.000" classname="com.testcase.LoginDemoApp">
    <skipped/>
  </testcase> <!-- Demo -->
  <testcase name="startSuite" time="0.000" classname="com.testcase.LoginDemoApp">
    <error type="java.io.FileNotFoundException" message="C:\Users\User\eclipse-workspace2\BasicFMM\src\test\java\com\config\config.properties (The system cannot find the path specified)">
      <![CDATA[java.io.FileNotFoundException: C:\Users\User\eclipse-workspace2\BasicFMM\src\test\java\com\config\config.properties (The system cannot find the path specified)
	at java.io.FileInputStream.open0(Native Method)
	at java.io.FileInputStream.open(FileInputStream.java:195)
	at java.io.FileInputStream.<init>(FileInputStream.java:138)
	at com.utility.ConfigFileReader.<init>(ConfigFileReader.java:18)
	at com.pages.BaseClass.startSuite(BaseClass.java:34)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:515)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:217)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:144)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:326)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
    </error>
  </testcase> <!-- startSuite -->
  <testcase name="loginAppAgain" time="0.000" classname="com.testcase.LoginDemoApp">
    <skipped/>
  </testcase> <!-- loginAppAgain -->
  <testcase name="loginApp" time="0.001" classname="com.testcase.LoginDemoApp">
    <skipped/>
  </testcase> <!-- loginApp -->
</testsuite> <!-- com.testcase.LoginDemoApp -->
